.github/ISSUE_TEMPLATE/minor_release.md

```diff
@@ -35,6 +35,7 @@ Change the `master` branch to the next MINOR+1 version:
 - [ ] Bump the `bump_dev_MAJOR_MINOR+1` branch to the next development MINOR+1 version.
   - Change the `Cargo.toml` version value to `MAJOR.MINOR+1.0`.
   - The commit message should be "Bump version to MAJOR.MINOR+1.0".
+- [ ] Update `CHANGELOG.md` file.
```

Comment:
Maybe this should go before "  - The commit message should be "Bump version to MAJOR.MINOR+1.0"." ? So that the `CHANGELOG.md` change goes in the bump commit:
```
- [ ] Bump the `bump_dev_MAJOR_MINOR+1` branch to the next development MINOR+1 version.
  - Change the `Cargo.toml` version value to `MAJOR.MINOR+1.0`.
  - Update `CHANGELOG.md` file.
  - The commit message should be "Bump version to MAJOR.MINOR+1.0".
```

Reply:
Yes as you have it makes much more sense, fixed.